contracts/standard/arbitration/composed-arbitrable/example/ArbitrableTokenList.sol

```diff
@@ -0,0 +1,594 @@
+
+        item.challengeReward = 0; // Clear challengeReward once a dispute is resolved.
+        item.lastAction = now;
+        agreement.parties[0].send(item.balance); // Deliberate use of send in order to not block the contract in case of reverting fallback.
+        item.balance = 0;
+
+        emit ItemStatusChange(agreement.parties[0], agreement.parties[1], _tokenID, item.status, agreement.disputed);
+    }
+
+    /** @dev Changes the `challengeReward` storage variable.
+     *  @param _challengeReward The new `stake` storage variable.
+     */
+    function changeChallengeReward(uint _challengeReward) public {
+        require(msg.sender == challengeRewardGovernor, "The caller is not the fee governor.");
+        challengeReward = _challengeReward;
+    }
+
+    /** @dev Changes the `challengeRewardGovernor` storage variable.
+     *  @param _challengeRewardGovernor The new `challengeRewardGovernor` storage variable.
+     */
+    function changeChallengeRewardGovernor(address _challengeRewardGovernor) public {
+        require(msg.sender == challengeRewardGovernor, "The caller is not the fee governor.");
+        challengeRewardGovernor = _challengeRewardGovernor;
+    }
+
+    /* Public Views */
+
+    /**
+     *  @dev Returns the latest agreement for an item
+     *  @param _tokenID The tokenID of the item to check.
+     *  @return The latest agreementID
+     */
+    function latestAgreementId(bytes32 _tokenID) public view returns (bytes32) {
+        return keccak256(abi.encodePacked(_tokenID, itemIDToAgreementCount[_tokenID]));
+    }
+
+    /**
+     *  @dev Return true if the item is allowed.
+     *  We consider the item to be in the list if its status is contested and it has not won a dispute previously.
+     *  @param _tokenID The tokenID of the item to check.
+     *  @return allowed True if the item is allowed, false otherwise.
+     */
+    function isPermitted(bytes32 _tokenID) public view returns (bool allowed) {
+        Item storage item = items[_tokenID];
+        Agreement storage latestAgreement = agreements[latestAgreementId(_tokenID)];
+        bool _excluded = item.status <= ItemStatus.Resubmitted ||
+            (item.status == ItemStatus.PreventiveClearingRequested && !latestAgreement.disputed);
+        return !_excluded;
+    }
+
+    /* Internal */
+
+    /** @dev Extends parent to use counter identify agreements.
+     *  @param _tokenID The item id.
+     *  @param _metaEvidence The meta evidence of the agreement.
+     *  @param _parties The `parties` value of the agreement.
+     *  @param _numberOfChoices The `numberOfChoices` value of the agreement.
+     *  @param _extraData The `extraData` value of the agreement.
+     *  @param _arbitrationFeesWaitingTime The `arbitrationFeesWaitingTime` value of the agreement.
+     *  @param _arbitrator The `arbitrator` value of the agreement.
+     */
+    function _createAgreement(
+        bytes32 _tokenID,
```

Comment:
Like with constructors, it's easier to read if the new params of an override are at the end.